wolfssl/ssl.h

```diff
@@ -530,6 +530,10 @@ struct WOLFSSL_BIO {
 #ifdef HAVE_EX_DATA
     WOLFSSL_CRYPTO_EX_DATA ex_data;
 #endif
+#if defined(OPENSSL_ALL) || defined(OPENSSL_EXTRA)
+    wolfSSL_Mutex    refMutex;  /* ref count mutex */
```

Comment:
Please guard this mutex variable with `!SINGLE_THREADED`.

Reply:
Fixed